doc/api/util.md

```diff
@@ -946,6 +946,161 @@ Otherwise, returns `false`.
 See [`assert.deepStrictEqual()`][] for more information about deep strict
 equality.
 
+## `util.parseArgs([argv][, options])`
+<!-- YAML
+added: REPLACEME
+-->
+
+* `argv` {string[]|Object} (Optional) Array of argument strings; defaults
+  to [`process.argv.slice(2)`](process_argv). If an Object, the default is used,
+  and this parameter is considered to be the `options` parameter.
+* `options` {Object} (Optional) The `options` parameter is an
+  object supporting the following properties:
+  * `optionsWithValue` {string[]|string} (Optional) One or more argument
```

Comment:
This is called `optionsWithValue`, implying that there are options without values, whereas the definition further below defines options to always have a value.